dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -169,4 +170,35 @@ utilisationReportsRouter
   .route('/reconciliation-details/:reportId')
   .get(validation.sqlIdValidation('reportId'), handleExpressValidatorResult, getUtilisationReportReconciliationDetailsById);
 
+/**
+ * @openapi
+ * /utilisation-reports/:bankId/:year:
+ *   get:
+ *     summary: |
+ *       Utilisation reports for the specified submission
+ *       year and bank. This includes status and details of
+ *       reports that have been submitted.
+ *     tags: [UtilisationReport]
+ *     description: |
+ *       Utilisation reports for the specified submission
+ *       year and bank. This includes status and details of
+ *       reports that have been submitted.
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *               items:
+ *                 $ref: '#/definitions/UtilisationReportReportReconciliationSummaryItem'
+ *       400:
+ *         description: Bad request
+ *       500:
+ *         description: Internal Server Error
+ */
+utilisationReportsRouter
+  .route('/:bankId/:year')
+  .get(validation.bankIdValidation, validation.yearValidation('year'), handleExpressValidatorResult, getUtilisationReportsByBankIdAndYear);
```

Comment:
In a similar vein of thought to the comment on the path, I think it would be good if the controller name indicated that we are returning reconciliation summaries rather than just reports as I made false assumptions on my first reading that it was returning full utilisation report objects.

Reply:
Moving to `bank-routes`